Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:lts-buster-slim
+
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+
+COPY . .
+
+EXPOSE 3000
+CMD [ "npm", "start" ]
```

Comment:
Instead of calling `npm`, we can use `next` as an entry point directly; it's at `node_modules/.bin/next`.

Reply:
(better for signal handling; possibly invoke the script through `node`)